src/features/Navigation/MobileDropdown/MobileNavItem.tsx

```diff
@@ -49,3 +52,14 @@ const LinkText = styled(Text)<{ isCurrent: boolean }>`
       text-underline-offset: 4px;
     `}
 `;
+
+const NotificationCircle = styled.div<{ isCurrent: boolean }>`
```

Comment:
The name isCurrent could be more explicit I think. Maybe isCurrentLocation?

Reply:
Good suggestion, I'll change this!